hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/OMBucketDeleteResponse.java

```diff
@@ -0,0 +1,57 @@
+/**
```

Comment:
Same here. This should be moved to a subsequent patch right?

Reply:
Added them for testing purpose to test DoubleBuffer Implementation.
Actual usage and the classes will be changed further according to needs of HA in further Jira.

In this PR #827 handled bucket related operations.